docker-compose-2pc.yml

```diff
@@ -7,7 +7,7 @@ services:
     image: opencbdc-tx
     tty: true
     restart: always
-    command: ./build/src/uhs/twophase/sentinel_2pc/sentineld-2pc 2pc-compose.cfg 0
+    command: ./build/src/uhs/twophase/sentinel_2pc/sentineld-2pc ./config/general/2pc-compose.cfg 0
```

Comment:
For all the `yml` files, this is a needed change and good; but it looks like this PR doesn't currently update the README's walk-through. We should make sure the documentation is fully-updated as well since the config paths will all change.

Reply:
Updated all the config file paths in the README.md. Thought about adding a section in the README regarding this new config file generation capability, but i think that may be premature at this moment since there isn't exactly a way to utilize the new config files. So adding it may confuse people before there is an actual integration test that utilizes the more complicated generated config files. What do you think?

NEEDS FOLLOW UP